frontend/artillery-load-test.yml

```diff
@@ -0,0 +1,49 @@
+config:
+  target: https://beta.grants.gov
```

Comment:
It feels a little awkward leaving this in for now, as we aren't retaining that URL by design. If you wanted to include just the dev URL, you could replace with:

```yml
  target: https://frontend-dev-143463875.us-east-1.elb.amazonaws.com
  tls:
    rejectUnauthorized: false
 ```

Blank or a placeholder could work too.